gtda/homology/grids.py

```diff
@@ -43,6 +49,11 @@ class CubicalPersistence(BaseEstimator, TransformerMixin):
         in a :obj:`joblib.parallel_backend` context. ``-1`` means using all
         processors.
 
+    Attributes
+    ----------
+    infinity_values_ : float or None, default : ``None``
```

Comment:
In what sense does an attribute have a default value? We don't do that elsewhere in the library AFAIK

Reply:
It's a mistake!